code/datums/actions/mobs/charge.dm

```diff
@@ -227,14 +227,19 @@
 	id = "tired_post_charge"
 	duration = 1 SECONDS
 	alert_type = null
+	var/tired_movespeed = /datum/movespeed_modifier/status_effect/tired_post_charge
 
 /datum/status_effect/tired_post_charge/on_apply()
 	. = ..()
-	owner.add_movespeed_modifier(/datum/movespeed_modifier/status_effect/tired_post_charge)
+	owner.add_movespeed_modifier(tired_movespeed)
 
 /datum/status_effect/tired_post_charge/on_remove()
 	. = ..()
-	owner.remove_movespeed_modifier(/datum/movespeed_modifier/status_effect/tired_post_charge)
+	owner.remove_movespeed_modifier(tired_movespeed)
+
+/datum/status_effect/tired_post_charge/easy
```

Comment:
"easy" is a little bit of a confusing word to use here as a subtype, not sure what it is trying to communicate?

Reply:
Any suggestion? "soft" perhaps?

Reply:
`reduced`, `lesser`, or `minor`? at least if I am reading it right